doc/usage/custom-ca.md

```diff
@@ -31,7 +31,7 @@ Steps of what happens with the custom cacert in k8s-cluster-api-provider:
    directory `~/cluster-defaults/${cloud_provider}-cacert` by OpenTofu
 3. During the management server bootstrap process cacert is injected to
    the *~/cluster-defaults/cluster-template.yaml* to *KubeadmControlPlane* and *KubeadmConfigTemplate* files
```

Comment:
```suggestion
   the `~/cluster-defaults/cluster-template.yaml` to *KubeadmControlPlane* and *KubeadmConfigTemplate* files
```

Reply:
I can't suggest these changes everywhere because Github seems to permit this only within the vicinity of your changes, but I would propose to apply similar changes throughout, so we don't switch between emphasis and code for the same purpose.